libvirt/domain.go

```diff
@@ -270,17 +258,28 @@ func setCoreOSIgnition(d *schema.ResourceData, domainDef *libvirtxml.Domain, vir
 				}
 			}
 		case "s390", "s390x", "ppc64", "ppc64le":
-			// System Z and PowerPC do not support any of the same pass-through
-			// mechanisms as Ignition. As a temporary workaround, the OpenStack
-			// Config Drive can be used instead. The Ignition volume already
-			// contains a Config Drive at this point.
-
-			disk, err := newDiskForCloudInit(virConn, ignitionKey, arch)
-			if err != nil {
-				return err
+			// System Z and PowerPC do not support the Firmware Configuration
+			// device. After a discussion about the best way to support a similar
+			// method for qemu in https://github.com/coreos/ignition/issues/928,
+			// decided on creating a virtio-blk device with a serial of ignition
+			// which contains the ignition config and have ignition support for
+			// reading from the device which landed in https://github.com/coreos/ignition/pull/936
+			domainDef.QEMUCommandline = &libvirtxml.DomainQEMUCommandline{
+				Args: []libvirtxml.DomainQEMUCommandlineArg{
```

Comment:
Hmm, not familiar with these golang libvirt bindings, though it looks like we can use [DomainDisk](https://godoc.org/github.com/libvirt/libvirt-go-xml#DomainDisk) and set the Serial member there.

Reply:
done.